Orange/widgets/visualize/owscatterplot.py

```diff
@@ -219,6 +224,11 @@ def reset_graph_data(self, *_):
     def set_data(self, data):
         self.information(1)
         self.__timer.stop()
+        for attr in data.domain:
+            if BasicStats(data[:, attr], attr).nans:
```

Comment:
@kernc : but the warnings should be added for attributes which contain even 1 nan. I don't think it is wrong.